### PR TITLE
adds notes for 4.9.6 release

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes to 4.9.6 release for Nov. 10

- Applies to `enterprise-4.9` only
- [Preview link](https://deploy-preview-38543--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-6)

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2110,3 +2110,35 @@ link:https://access.redhat.com/solutions/6465271[{product-title} 4.9.5 container
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-9-6"]
+=== RHBA-2021:4119 - {product-title} 4.9.6 bug fix and security update
+
+Issued: 2021-11-11
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2117]: This date needs to change to 2021-11-10. I will change after the input from peer review.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2110,3 +2110,35 @@ link:https://access.redhat.com/solutions/6465271[{product-title} 4.9.5 container
...
+
+{product-title} release 4.9.6 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4119[RHBA-2021:4119] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:4118[RHSA-2021:4118] advisory.
```
Comment (ctauchen) [inline release_notes/ocp-4-9-release-notes.adoc:2119]:
```suggestion
{product-title} release 4.9.6, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4119[RHBA-2021:4119] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:4118[RHSA-2021:4118] advisory.
```

Comment (opayne1) [reply release_notes/ocp-4-9-release-notes.adoc:2119]: Darn! I always forget that bit. Thanks!

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2110,3 +2110,35 @@ link:https://access.redhat.com/solutions/6465271[{product-title} 4.9.5 container
...
+==== Known Issues
+
+* The current opt-in obfuscation will not work on clusters with OVN because the `hostsubnets.network.openshift.io` is not currently on OVN clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014633[BZ#2014633])
+
+[id="ocp-4-9-6-bug-fixes"]
+==== Bug fixes
+
+* Previously, a bug in the lock implementation for the `nmstate-handler` pod caused multiple nodes to gain control. This update fixes the lock implementation so that only one node is in control of the lock. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954309[*BZ#1954309*])
+
+* Previously, OpenStack flavor validation accepted flavors not meeting the RAM requirements using the wrong unit. With this update, the correct unit is used for comparing minimum RAM against value returned by OpenStack. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2009787[*BZ#2009787])
```
Comment (skrthomas) [inline release_notes/ocp-4-9-release-notes.adoc:2135]: From here down, the BZ link text needs another '*' to bold

**Review by skrthomas — COMMENTED** [review]
Just one formatting thing. LGTM otherwise :)

**Pull request merged**